### DISCUSSION TITLE
is it possible to use prisma db pull to generate json file ?

### DISCUSSION THREAD

**Discussion by xiangnanscu** [prisma/prisma#14460] [Q&A]
rt

**Comment by austincrim** [comment]
Hey @xiangnanscu 👋,

This is not possible today, but depending on what format you want the JSON file to be in, you could look at [this community project](https://github.com/valentinpalkovic/prisma-json-schema-generator).

If you are looking for something else, feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with more details!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
